src/commonmark-rules.js

```diff
@@ -111,11 +111,25 @@ rules.fencedCodeBlock = {
   replacement: function (content, node, options) {
     var className = node.firstChild.className || ''
     var language = (className.match(/language-(\S+)/) || [null, ''])[1]
+    var code = node.firstChild.textContent
+
+    var fenceChar = options.fence === '~~~' ? '~' : '`'
```

Comment:
A _really_ minor one! To decouple the fence characters from the replacement function:
 
```suggestion
    var fenceChar = options.fence.charAt(0)
```

(A contrived example: say `???` is introduced as another fence, we'd have to rewrite to handle that.)